cmd/database/database_create.go

```diff
@@ -87,45 +88,51 @@ var dbCreateCmd = &cobra.Command{
 
 		dbVersions, err := client.ListDBVersions()
 		if err != nil {
-			utility.Error("Unable to list database versions %s", err)
+			utility.Error("Failed to fetch database versions: %s", err)
 			os.Exit(1)
 		}
 
-		if software == "" {
-			software = "MySQL"
-		}
+		software = strings.ToLower(software)
+		softwareIsValid := false
+		softwareVersionIsValid := false
 
-		if softwareVersion == "" && software == "MySQL" {
-			softwareVersion = "8.0"
+		validSoftwares := map[string][]string{
+			"mysql":      {"mysql", "my", "sql"},
```

Comment:
I don't think we should allow "my" and "sql"

Reply:
https://github.com/civo/cli/pull/413/commits/b672f2650e5875f60f871107e45edb5862036c5a